english/expelliarmus/expelliarmus.md

```diff
@@ -0,0 +1,4 @@
+Use this disarming charm, also known as the Expelliarmus spell
+as a defensive tactic to force the marshmallow to release you.
+This has worked for Harry Pothead but may not for you.
+Given your current level of intoxication. In that case you may just need to take the [red pill](https://github.com/udacity/create-your-own-adventure/blob/master/english/red-pill/red-pill.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.